cmake/app/boilerplate.cmake

```diff
@@ -307,38 +308,56 @@ foreach(root ${BOARD_ROOT})
         )
         list(APPEND
           shield_dts_fixups
-          ${shield_dir}/${s}/dts_fixup.h
+          ${shield_dir}/${s_dir}/dts_fixup.h
         )
+
+        # search for shield/boards/board.overlay file
```

Comment:
I think if we're going to move the existing searches inside the `if (NOT _idx EQUAL -1)` block, then we do that in a single patch and explain the move.  Then a 2nd patch to use s_dir instead of s.

Reply:
ok, split up the commits.